helm/cas-provision/templates/deployer/deployerRole.yaml

```
@@ -0,0 +1,181 @@
+# Permissions removed during migration:
+# {APIGroups:[""], Resources:["pods/log"], Verbs:["create" "delete" "deletecollection" "patch" "update"]}
+
+apiVersion: authorization.openshift.io/v1
+kind: Role
+metadata:
```

This should be added to every template in the chart:
```suggestion
metadata:
  labels:
{{ include "cas-provision.labels" . | indent 4 }}
```